src/Materials/Node/nodeMaterial.ts

```diff
@@ -151,10 +151,10 @@ export class NodeMaterial extends PushMaterial {
     /** Gets or sets a boolean indicating that node materials should not deserialize textures from json / snippet content */
     public static IgnoreTexturesAtLoadTime = false;
 
-    private BJSNODEMATERIALEDITOR = this._getGlobalNodeMaterialEditor();
+    private BJSNODEMATERIALEDITOR = NodeMaterial._getGlobalNodeMaterialEditor();
 
     /** Get the inspector from bundle or global */
-    private _getGlobalNodeMaterialEditor(): any {
+    static _getGlobalNodeMaterialEditor(): any {

```

Comment:
can you make it private static and then rename it to _**G**etGlobalNodeMaterialEditor ?